src/azure-cli-testsdk/azure/cli/testsdk/preparers.py

```diff
@@ -115,7 +115,7 @@ def remove_resource(self, name, **kwargs):
 # pylint: disable=too-many-instance-attributes
 class StorageAccountPreparer(NoTrafficRecordingPreparer, SingleValueReplacer):
     def __init__(self, name_prefix='clitest', sku='Standard_LRS', location='westus', kind='Storage',
-                 allow_blob_public_access=False, hns=False, length=24,
+                 allow_blob_public_access=False, allow_shared_key_access=True, hns=False, length=24,
```

Comment:
Maybe we should default this to `False` since it's compliant with company security requirements

Reply:
Keeping default as `None` until service changes the default to `False` as well.